alibi_detect/saving/schemas.py

```diff
@@ -889,6 +867,314 @@ class LearnedKernelDriftConfigResolved(DriftDetectorConfigResolved):
     train_kwargs: Optional[dict] = None
     dataset: Optional[Callable] = None
     device: Optional[Literal['cpu', 'cuda']] = None
+    dataloader: Optional[Callable] = None  # TODO: placeholder, will need to be updated for pytorch implementation
+
+
+class ContextMMDDriftConfig(DriftDetectorConfig):
+    """
+    Unresolved schema for the
+    `ContextMMDDrift <https://docs.seldon.io/projects/alibi-detect/en/stable/cd/methods/contextmmddrift.html>`_
+    detector.
+
+    Except for the `name` and `meta` fields, the fields match the detector's args and kwargs. Refer to the
+    :class:`~alibi_detect.cd.ContextMMDDrift` documentation for a description of each field.
+    """
+    p_val: float = .05
+    c_ref: str
+    preprocess_at_init: bool = True
+    update_ref: Optional[Dict[str, int]] = None
+    x_kernel: Optional[Union[str, KernelConfig]] = None
+    c_kernel: Optional[Union[str, KernelConfig]] = None
+    n_permutations: int = 100
+    prop_c_held: float = 0.25
+    n_folds: int = 5
+    batch_size: Optional[int] = 256
+    verbose: bool = False
+    device: Optional[Literal['cpu', 'cuda']] = None
+
+
+class ContextMMDDriftConfigResolved(DriftDetectorConfigResolved):
+    """
+    Resolved schema for the
+    `MMDDrift <https://docs.seldon.io/projects/alibi-detect/en/stable/cd/methods/mmddrift.html>`_ detector.
+
+    Except for the `name` and `meta` fields, the fields match the detector's args and kwargs. Refer to the
+    :class:`~alibi_detect.cd.MMDDrift` documentation for a description of each field.
+    """
+    p_val: float = .05
+    c_ref: np.ndarray
+    preprocess_at_init: bool = True
+    update_ref: Optional[Dict[str, int]] = None
+    x_kernel: Optional[Callable] = None
+    c_kernel: Optional[Callable] = None
+    n_permutations: int = 100
+    prop_c_held: float = 0.25
+    n_folds: int = 5
+    batch_size: Optional[int] = 256
+    verbose: bool = False
+    device: Optional[Literal['cpu', 'cuda']] = None
+
+
+class MMDDriftOnlineConfig(DriftDetectorConfig):
+    """
+    Unresolved schema for the
```

Comment:
Remove "unresolved" since you've done it for others..?

Reply:
See above.